apps/web/screens/AboutPage/AboutPage.tsx

```diff
@@ -160,7 +160,7 @@ const PageHeader: FC<PageHeaderProps> = ({ page }) => {
   return (
     <Background id={slice.id} theme={page.theme}>
       {!!mobileNavigation.length && (
-        <Box display={['block', 'block', 'block', 'none']} paddingBottom={4}>
+        <Hidden above="md">
```

Comment:
Are we removing padding bottom here? and is it not on the agenda to deprecate Hidden component?

Reply:
Padding bottom was just introducing spacing on top of the page and not affecting the DrawerMenu. Not sure what the point was.

I'll replace Hidden when/if Box supports hiddenAbove syntax.